src/responder/pam/pamsrv_gssapi.c

```diff
@@ -18,12 +18,14 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include <asm-generic/errno-base.h>
```

Comment:
Hi,

why is this `#include` needed? I do not get a compilation error if I remove it and imo it is already include via `errno.h`.

bye,
Sumit

Reply:
I don't know. It looks like an overly smart IDE I tried to use for this PR preparation. ;) Fixed.